pkg/controller/issueapi/issue.go

```diff
@@ -106,7 +106,9 @@ func (c *Controller) IssueMany(ctx context.Context, requests []*api.IssueCodeReq
 			wg.Add(1)
 			go func(request *api.IssueCodeRequest, r *IssueResult) {
 				defer wg.Done()
-				c.SendSMS(ctx, realm, smsProvider, smsSigner, keyID, request, r)
+				if err := c.SendSMS(ctx, realm, smsProvider, smsSigner, keyID, request, r); err != nil {
```

Comment:
@whaught double check me here. It looks like we might have been silently discarding the error returned from sending the SMS?

Reply:
Well this is interesting because we set r.ErrorReturn on the inner call. I might double check this with a followup